assets/eip-4910/eip-4910-reference-implementation/contracts/PaymentModule.sol

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: Unlicensed
```

Comment:
Must be compatible with CC0-1.0

Reply:
@SamWilsn Apache 2.0 ok of the reference implementation? .. It is marked as Apache 2.0 in the licensing file,

Reply:
@SamWilsn added Apache 2.0 to all the .sol files